### PR TITLE
v5.6.1

### PR DESCRIPTION
**Pull request by flaviendelangle**
I don't see anything worth highlighting for now.

### CODE REVIEW

**Review by cherniavskii — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,39 @@
...
+
+- [docs] Fix links to prevent duplicate search result (#4130) @siriwatknp
+- [docs] Fix outdated links to `localeTextConstants.ts` (#4080) @patilvishal755
+- [docs] Neglect e2e tests related to search (#4118) @siriwatknp
+- [docs] Use regex instead of specific url in e2e-website-tests (#4121) @siriwatknp
+
+### Core
+
+- [core] Enforce `noImplicitAny` (#4084) @cherniavskii
+- [core] Improve the pro support issue template (#4082) @oliviertassinari
```
Comment (m4theushw) [inline CHANGELOG.md:32]:
```suggestion
- [core] Improve the Pro support issue template (#4082) @oliviertassinari
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,39 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.6.1
+
+_Mar 10, 2022_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible.
+
```
Comment (m4theushw) [inline CHANGELOG.md:11]: I just merged #3848

The highlight is in the PR's description.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,39 @@
...
+### `@mui/x-data-grid@v5.6.1` / `@mui/x-data-grid-pro@v5.6.1`
+
+#### Changes
+
+- [DataGrid] Display column's filter icon if a filter is applied (#4120) @DanailH
+- [DataGrid] Do not loop through rows to compute top level rows count wen the tree is flat (#4081) @flaviendelangle
+- [DataGrid] Support extending built-in column types (#4114) @cherniavskii
+- [DataGrid] Use `buildWarning` and `wrapWithWarningOnCall` for deprecated methods and wrong usages (#4056) @flaviendelangle
```
Comment (alexfauquette) [inline CHANGELOG.md:19]:
```suggestion
- [core] Use `buildWarning` and `wrapWithWarningOnCall` for deprecated methods and wrong usages (#4056) @flaviendelangle
```

**Review by alexfauquette — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,47 @@
...
+  <DataGrid getRowSpacing={() => ({ top: 10, bottom: 10 })} />
+  ```
+
+  Check the [documentation](https://mui.com/components/data-grid/rows/#row-spacing) for more information.
+
+### `@mui/x-data-grid@v5.6.1` / `@mui/x-data-grid-pro@v5.6.1`
+
+#### Changes
+
+- [DataGrid] Display column's filter icon if a filter is applied (#4120) @DanailH
```
Comment (DanailH) [inline CHANGELOG.md:24]: That was my bad.

```suggestion
- [DataGrid] Aways show column's filter icon if a filter is applied (#4120) @DanailH
```

**Review by DanailH — APPROVED** [review]

**Review by cherniavskii — COMMENTED** [review]
Let's wait for #4149

**Pull request merged**